server/meshmodel/kubernetes/relationships/hierarchical_secret.json

```diff
@@ -0,0 +1,172 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+      "description": "A relationship that assigns configuration data for resources deployed in a cluster by environment variable.",
```

Comment:
@RipulHandoo isn't this the description for ConfigMap? Any changes for Secret?

Reply:
```suggestion
      "description": "Use this relationship to make sensitive information available to other resources, like your application, by environment variable.",
```

Reply:
Done.

Reply:
Done